requirements/V2_REQUIREMENTS.md

```diff
@@ -0,0 +1,74 @@
+
+### Smart Cities
+
+### Manufacturing
+
+### Energy
+
+### Transportation
+
+### Agriculture
+
+## Requirements
+
+### WoT Profiles Specification
+
+The WoT Profiles specification:
+1. MUST define a mechanism by which the author of a WoT Thing Description can denote that the Thing it describes conforms to a particular profile
+2. MUST define a registry of profiles to which WoT Things and WoT Consumers may conform
+3. SHOULD aim to keep the number of registered profiles as small as possible by rejecting profiles that significantly overlap in technologies or use cases, in order to minimise fragmentation on the Web of Things
+### Individual Profiles
+
+Individual profile documents:
+1. MUST specify an identifier (URI) which identifies the profile 
+2. SHOULD constrain a finite set of protocol bindings that a conformant Thing may use and that a conformant Consumer can be expected to support
+	1. MAY constrain conformant Things to always follow certain defaults defined in a protocol binding document
+3. SHOULD constrain a finite set of payload bindings that a conformant Thing may use and that a conformant Consumer can be expected to support
+	1. MAY constrain conformant Things to always follow certain default payload formats defined in a payload binding document
+	2. MAY constrain conformant Things to follow certain conventions such as date formats and error formats
+4. SHOULD constrain a finite set of security mechanisms that a conformant Thing may use and that a conformant Consumer can be expected to support
+5. SHOULD constrain a finite set of discovery mechanisms that a conformant Thing may use and that a conformant Consumer can be expected to support
+6. MAY constrain a finite set of link relation types that a conformant Thing is recommended to use, and how they should be interpreted by a WoT Consumer
+7. MAY constrain a finite set of semantic contexts that a conformant Thing is recommended to use and that a conformant Consumer can be expected to support
+8. SHOULD NOT directly extend the WoT Thing Description information model, which should be defined using TD context extensions
+9. SHOULD NOT define protocol behaviour, which should be defined in a protocol binding document
+10. SHOULD NOT define data payload formats, which should be specified in a payload binding document
+11. SHOULD NOT define discovery mechanisms, which should be defined in the WoT Discovery specification
+12. SHOULD NOT define security mechanisms, which should be defined in a protocol binding document
+13. SHOULD NOT describe an existing single-vendor IoT platform, which should be defined in a platform binding document
+14. MUST NOT include any assertions which would require a Thing Description of a conformant Web Thing to be non-conformant with the Thing Description 2.0 specification
+15. SHOULD NOT include any assertions which conflict with another Profile, so that a Thing may conform with multiple Profiles
```

Comment:
Do we want to allow a thing to describe something outside of the profile scope?

e.g. a thing may sport an http profile that covers only properties and all the properties then have an http form constrained by the profile, but it can also provide coap forms and event affordances in the same description. 

The consumer then may just ignore them or process them using the degraded consumption rules we should write down in the TD document.

Reply:
Yes, and there is a [note](https://w3c.github.io/wot-profile/#h-note-0) to this effect in the existing 1.0 specification.

I've added this note the end of the introduction.